include/glow/Base/Tensor.h

```diff
@@ -573,6 +573,9 @@ class Tensor final {
   /// Convert each element of this tensor to \p newTy.
   void convertToType(ElemKind newTy);
```

Comment:
could you update the comment on what's currently supported, in terms of typea -> typeb conversion